libafl_frida/src/utils.rs

```diff
@@ -228,38 +228,33 @@ pub(crate) fn frida_to_cs(
 /// Get the `base`, `idx`, `scale`, `disp` for each operand
 pub fn operand_details(operand: &Operand) -> Option<(X86Register, X86Register, u8, i32)> {
     match operand {
-        Operand::RegDeref(base) => {
```

Comment:
@s1341 can you see if that's all right?

Reply:
I don't have time to test at the moment. We'll see if it breaks things :| yolo, right?

Reply:
yoloo